usaspending_api/common/elasticsearch/client.py

```diff
@@ -48,7 +53,9 @@ def create_es_client():
         logger.error("Error creating the elasticsearch client: {}".format(e))
 
 
-def es_client_query(index, body, timeout="1m", retries=5):
+def es_client_query(
+    index: str = None, body: dict = None, timeout: str = "1m", retries: int = 5, search: Search = None
+) -> Union[Elasticsearch, Response]:
```

Comment:
Confused by the `Union` typehint. Seems like it would be something like `Optional[Union[dict, Response]]` (unsure if that is proper) since None is a possible return value

If desired, the response type of _es_search() can be defined as a new type, which could be utilized in other functions' I/O.

Reply:
Created a unique type to use instead. Also, helps to keep the line from being really long.